### PR TITLE
Make the weekbox an optional feature

### PR DESCRIPTION
**Pull request by vinceve**
The weekbox can now be configured in the options with a boolean value.

### CODE REVIEW

**Comment by eleazan** [comment]
:+1: !

I think it's correct... Only needs a litte documentation! Can you pr a little explanation? ;)

**Comment by vinceve** [comment]
I will do this later today! :-)

**Comment by vinceve** [comment]
Done.

File: js/calendar.js [hunk 1]
```diff
@@ -1054,7 +1056,9 @@ if(!String.prototype.formatNum) {
 
 		var week = $(document.createElement('div')).attr('id', 'cal-week-box');
```
Comment (Serhioromano) [inline js/calendar.js]: I think this line also have to be inside conditions because this is the box itself.

File: js/calendar.js [hunk 2]
```diff
@@ -1067,7 +1071,8 @@ if(!String.prototype.formatNum) {
 			.on('mouseleave', function() {
 				week.hide();
 			})
-		;
+			;
+		}
 
 		week.click(function() {
```
Comment (Serhioromano) [inline js/calendar.js]: And this block too should be inside condition.

**Comment by Serhioromano** [comment]
I know it has been a long time but I wanted to merge this. Unfortunately it cannot be automatically merged now after all changes. Can you update upstream and resubmit PR?

**Comment by vinceve** [comment]
Sorry I saw your comment now :)

**Comment by Serhioromano** [comment]
It is ok. The changes are in calendar right now.

**Pull request closed without merging**